announcements/announcements.py

```diff
@@ -1,32 +1,35 @@
 import domControl
-
+import crud_announcement
```

Comment:
from crud import announcement, crud_announcement does not exist

 or import crud.announcement would also be valid here

Reply:
crud_announcement is actually a file we fetch in the index.html for the announcements